packages/lib-user/src/components/shared/ProjectCard/ProjectCard.js

```diff
@@ -84,7 +86,9 @@ function ProjectCard ({
       forwardedAs='a'
       href={href}
       round='8px'
-      small={small}
+      cardFontSize={cardFontSize(size)}
+      height={`${cardWidth(size) * 14 / 11}px`}
```

Comment:
The ratio of every card is 11:14, so it can be used here to calculate height based on width 🤷‍♀️

Reply:
The card and background images sizes all look good to me!